### ISSUE TITLE
MAD and IQR readouts showing '?' when equal to zero

### ISSUE THREAD

**Issue by matthew-blackman** [phetsims/center-and-variability#210]
I noticed that the MAD and IQR readouts are showing '?' when the reading should be zero, even when there is enough data. I traced it back to `VariabilityAccordionBox.ts`, where there is the following type of guard for IQR and MAD:

```
const iqrReadoutValueProperty = new DerivedProperty( [ iqrValueProperty ], iqrValue => {
          return iqrValue ? `${iqrValue}` : '?';
        } );
```

This is being done differently for the `rangeReadoutValueProperty`, which is displaying correctly.

**Comment by marlitas** [comment]
It's probably because `0` reads as falsy and I didn't think about that while refactoring at one point. My bad. I'll take it.

**Comment by marlitas** [comment]
Fixed above. However I did notice that the IQR value is reading as `0` even when the sim says "Need at least 5 kicks"

<img width="1316" alt="image" src="https://github.com/phetsims/center-and-variability/assets/42476338/d6101b10-b6f0-40ee-982b-51f8afe888a0">
I'm guessing:

- [ ] the IQR value needs to be reset to null when kicks are cleared out
- [ ] the IQR value remains at null until there is enough data to calculate.

Not as sure about the second suggestion in terms of pedagogy and design, or if that is indeed correct in how that would calculate if there isn't enough data.
